week-4/implement/card-validator.js

```diff
@@ -0,0 +1,36 @@
+// Number must be 16 digits, all of them must be numbers.
+// - You must have at least two different digits represented (all of the digits cannot be the same).
+// - The final digit must be even.
+// - The sum of all the digits must be greater than 16.
+function addingCard(card) {
+    // typeof card === 'number' ;
+    const num = String(card).split("").map(Number);
+    
+    let add = 0;
+
+    for (let i = 0; i < num.length - 1; i += 2) {
+        add += num[i] + num[i + 1];
+    }
```

Comment:
I think this loop is slightly more complicated than it needs to be. If we go from `i == 0` to `i < num.length`, we will already visit each number once. Then we only need to add the current element, instead of the current element and the one after that